src/Compiler/Backend/GLSL/GLSLConverter.cpp

```diff
@@ -813,6 +813,9 @@ void GLSLConverter::ConvertIntrinsicCall(CallExpr* ast)
         case Intrinsic::Texture_SampleLevel_5:
             ConvertIntrinsicCallTextureSampleLevel(ast);
             break;
+        case Intrinsic::Image_Store:
```

Comment:
Why is this handeld for conversion in the `GLSLConverter`, since `Image_Store` is GLSL only?
I.e. it is generated during the conversion itself, so why should it be handeled for conversion again?